src/main/java/Exceptions/InvalidKeyException.java

```
@@ -0,0 +1,8 @@
+package Exceptions;
+
+public class InvalidKeyException extends Exception{
+
+    public InvalidKeyException(String message, Throwable cause) {
```

what if i want to throw this exception without throwable cause?